static-container-registry/static-container-registry.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+
+import sys
+import os.path
+import json
+import hashlib
+import logging
+import argparse
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+CONSTANTS = '''
```

Comment:
this duplicates these locations  (/v2, /v2/, @404_tag) on multiple solutions deployment and nginx will crash on it

Reply:
Thanks, will fix (there's an option to pass to the script IIRC)

Reply:
Oh actually not, need to fix upstream, I'll do a PR and include the change here in the meantime.